supabase-ws-flutter-mobile/lib/ui/home/home_screen.dart

```diff
@@ -0,0 +1,255 @@
+          appBar: AppBar(
+            leading: Assets.logoAndName.image(width: 143.w),
+            leadingWidth: 160.w,
+            backgroundColor: context.theme.colors.background.shade500,
+            actions: [
+              IconButton(
+                icon: const Icon(Icons.more_vert),
+                color: context.theme.colors.textColor.shade100,
+                //TODO: add logic
+                onPressed: () => {},
+              ),
+            ],
+          ),
+          body: Column(
+            children: [
+              state.messages.isEmpty
+                  ? Expanded(
+                      child: Container(
+                        decoration: BoxDecoration(
+                          color: context.theme.colors.background.shade400,
+                        ),
+                        child: Row(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          children: [
+                            Column(
+                              mainAxisAlignment: MainAxisAlignment.center,
+                              children: [
+                                Assets.sendGreen.image(height: 40.h),
+                                Text(
+                                  context.localizations.home_empty_state,
+                                  textAlign: TextAlign.center,
+                                  style: context.theme.textStyles.titleMedium
+                                      ?.copyWith(
+                                    color:
+                                        context.theme.colors.primary.shade300,
+                                  ),
+                                ),
+                              ],
+                            ),
+                          ],
+                        ),
+                      ),
+                    )
+                  : Expanded(
+                      child: Container(
+                        decoration: BoxDecoration(
+                          color: context.theme.colors.background.shade400,
+                        ),
+                        child: ListView.builder(
+                          itemCount: state.messages.length,
+                          itemBuilder: (context, index) {
+                            final userMessage = state.messages[index];
+                            return _MessageWidget(userMessage: userMessage);
+                          },
+                        ),
+                      ),
+                    ),
+              const _TextFieldSection(),
+              SizedBox(height: 24.h),
+            ],
+          ),
+        ),
+      );
+}
+
+class _TextFieldSection extends StatefulWidget {
+  const _TextFieldSection();
+
+  @override
+  State<_TextFieldSection> createState() => _TextFieldSectionState();
+}
+
+class _TextFieldSectionState extends State<_TextFieldSection> {
+  final textController = TextEditingController();
```

Comment:
private and shouldn't be defined in the previous class? 🤔 What happen when the messages changes?